woocommerce-pdf-invoices-packingslips.php

```diff
@@ -333,20 +333,34 @@ public function nginx_detected()
 		}
 		$tmp_path = $this->main->get_tmp_path('attachments');
 		$server_software   = isset( $_SERVER['SERVER_SOFTWARE'] ) ? sanitize_text_field( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) ) : false;
```

Comment:
If you're going to perform `stristr`  on this it's better to fall back to `''` instead of `false`.